Library/Formula/mandoc.rb

```diff
@@ -0,0 +1,74 @@
+class Mandoc < Formula
+  homepage "http://mdocml.bsd.lv"
+  url "http://mdocml.bsd.lv/snapshots/mdocml-1.13.3.tar.gz"
+  sha256 "23ccab4800d50bf4c327979af5d4aa1a6a2dc490789cb67c4c3ac1bd40b8cad8"
+
+  head "anoncvs@mdocml.bsd.lv:/cvs", :module => "mdocml", :using => :cvs
+
+  depends_on "sqlite" if build.with? "db"
+
+  option "without-db", "Only install the mandoc/demandoc utilities."
```

Comment:
Can we make this `without-sqlite` instead and then just make the `sqlite` dep `=> :optional`? Conditional deps are problematic.